src/pudl/metadata/resources/ferc1_eia_record_linkage.py

```diff
@@ -124,6 +127,34 @@
         "etl_group": "outputs",
         "field_namespace": "eia",
     },
+    "out_eia__yearly_assn_plant_parts_plant_gen": {
+        "description": """In order to easily determine what generator records are associated with every
+plant part record, we made this association table. This table associates every plant part
+record (identified as ``record_id_eia``) from the ``out_eia__yearly_plant_parts`` table to the associated
+'plant_gen' records (identified as ``record_id_eia_plant_gen``). All of the columns in this table
+besides ``record_id_eia`` are attributes of the plant_gen record and have a suffix of '_plant_gen'.""",
+        "schema": {
+            "fields": [
+                "record_id_eia",
+                "record_id_eia_plant_gen",
+                "report_date_plant_gen",
+                "operational_status_pudl_plant_gen",
+                "energy_source_code_1_plant_gen",
+                "ferc1_generator_agg_id_plant_gen",
+                "prime_mover_code_plant_gen",
+                "unit_id_pudl_plant_gen",
+                "technology_description_plant_gen",
+                "ferc_acct_name_plant_gen",
+                "plant_id_eia_plant_gen",
+                "generator_operating_year_plant_gen",
+                "generator_id_plant_gen",
+            ],
+            "primary_key": ["record_id_eia", "record_id_eia_plant_gen"],
```

Comment:
just verifying that report date doesn't need to be a PK?

Reply:
good question! but no the report date doesn't need to be in there because there is a year element in the record_id_eia*